src/index.html

```diff
@@ -1,17 +1,126 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.erokuapp.com/create-application" 
+    onsubmit="onSubmit()"
+    method="POST">
+      <fieldset class="form-block">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input type="text" 
+          name="surname" 
+          autocomplete="off">
```

Comment:
```suggestion
          <input
            type="text"
            name="surname" 
            autocomplete="off"
          >
```
неправильное форматирование тегов. Закрывающая скобка должна быть на отдельной строке, если атрибутов больше двух - то каждый на новой строке.
Исправь форматирование всех тегов

Reply:
I made chenges